# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg/biarch-wo-multilib.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

This is an attempt to get a biarch compiler without full multilib stuff.

  - Rene Rebe <rene@exactcode.de>

--- xc/config/cf/linux.cf.orig	2005-01-17 22:06:58.661903120 +0100
+++ xc/config/cf/linux.cf	2005-01-17 22:11:38.037431584 +0100
@@ -540,6 +540,8 @@
 
 #define MkdirHierCmd		mkdir -p
 
+#define HaveLib64     NO
+
 #ifndef HaveLib64
 # if defined (AMD64Architecture) || defined (s390xArchitecture) || defined (Ppc64Architecture)
 #  define HaveLib64	YES
